openpype/hosts/blender/plugins/publish/extract_blend.py

```diff
@@ -160,6 +161,31 @@ def _get_used_images(
             if users & datablocks
         }
 
+    def _process_resource_data(
```

Comment:
'data' is too general. Maybe
```suggestion
    def _process_resource_file(
```